qiskit/test/reference_pulses.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
You could rebase on my pulse library PR if would like to use the tested pulses within that.